packages/rollup-services/exec/build_and_run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# build_and_run.sh -- accepts a command to run after postgres connection succeeds
+# conditionally builds based on REBUILD env var and then calls wait_for_postgres_and_geth.sh
+
+# Exits if any command fails
+set -e
+
+cmd=$@
+
+ROOT_DIR=../../..
```

Comment:
`ROOT_DIR` is the root of the git repo?

Depending on how this script is called, this may or may not be useful. If this file ever moves, then you'll have to update the `ROOT_DIR` value here. This bash oneliner is a way to get the `cwd` that the script is called from that i normally use to get the root dir. But this is a monorepo so its different than my normal usecase

```bash
ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
```

Reply:
Good to know about the `BASH_SOURCE` var!

Yes, `ROOT_DIR` in this context is the root of the repo, not the user's root dir. I can change the name if this is confusing. We'll want to be relative to the repo because that's the only thing we can make safe assumptions about regardless of where it's running.

Reply:
My main confusion around `ROOT_DIR` was the root of the monorepo or the root of the package. I think its fine to leave it the way that it is.